websites/K/Kemono.Party/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "I Am Lel#5090",
+		"id": "850319718920224798"
+	},
+	"service": "Kemono.Party",
+	"description": {
+		"en": "Welcome to Kemono, a scraper for Patreon, Pixiv Fanbox, SubscribeStar, Gumroad, Discord, DLsite, and Fantia."
+	},
+	"url": "kemono.party",
+	"regExp": "([a-z0-9-]+[.])*kemono[.]party[/]",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/mwpEkf3.png",
```

Comment:
Logos should be [512x512px squares](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).